<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="tandl-10" name="Test 2" tests="1" failures="1" timestamp="10 Aug 2015 12:34:41 GMT" time="15.169" errors="0">
  <testcase name="verifySubmenus" time="15.153" classname="com.sweety.testcases.VerifySubmenus">
    <failure type="java.lang.Throwable" message="Multiple failures (2):nnFailure 1 of 2:njava.lang.AssertionError: expected [Level Reports] but found [Level Entries]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.sweety.util.Keywords.assertEquals(Keywords.java:946)
	at com.sweety.util.Keywords.executeKeywords(Keywords.java:152)
	at com.sweety.testcases.VerifySubmenus.verifySubmenus(VerifySubmenus.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
nnFailure 2 of 2:njava.lang.AssertionError: Fail">
      <![CDATA[java.lang.Throwable: Multiple failures (2):nnFailure 1 of 2:njava.lang.AssertionError: expected [Level Reports] but found [Level Entries]
	at com.sweety.util.Keywords.assertEquals(Keywords.java:946)
	at com.sweety.util.Keywords.executeKeywords(Keywords.java:152)
	at com.sweety.testcases.VerifySubmenus.verifySubmenus(VerifySubmenus.java:35)
nnFailure 2 of 2:njava.lang.AssertionError: Fail
	at com.sweety.util.Keywords.executeKeywords(Keywords.java:212)
	at com.sweety.testcases.VerifySubmenus.verifySubmenus(VerifySubmenus.java:35)
... Removed 60 stack frames]]>
    </failure>
  </testcase> <!-- verifySubmenus -->
</testsuite> <!-- Test 2 -->
